Assets/BossRoom/Scripts/Shared/Net/GameNetPortal.cs

```diff
@@ -122,6 +144,16 @@ private void RegisterClientMessageHandlers()
                     ConnectFinished?.Invoke(status);
                 }
             });
+
+            MLAPI.Messaging.CustomMessagingManager.RegisterNamedMessageHandler("S2C_SetDisconnectReason", (senderClientId, stream) =>

```

Comment:
Same here for abbreviations & underscores.

Reply:
Also should this be an explicit method? Could it suffer the same problem with unregistering of anonymous methods?

Reply:
I don't think it makes any difference in this particular case, because we're not just plugging it into a C# event. The MLAPI signature is to register and unregister by string method name, which I can do equally well with an anonymous or regular method.